bin/postgres-ha/bootstrap/post-bootstrap.sh

```diff
@@ -20,29 +20,33 @@ enable_debugging
 
 echo_info "postgres-ha post-bootstrap starting"
 
-# Run either a custom or the defaul setup.sql file
-if [[ -f "/pgconf/setup.sql" ]]
+if [[ "${PGHA_BOOTSTRAP_METHOD}" == "initdb" ]]
 then
-    echo_info "Using custom setup.sql"
-    cp "/pgconf/setup.sql" "/tmp"
+    # Run either a custom or the defaul setup.sql file
+    if [[ -f "/pgconf/setup.sql" ]]
+    then
+        echo_info "Using custom setup.sql"
+        setup_file="/opt/cpm/bin/sql/setup.sql"
+    else
+        echo_info "Using default setup.sql"
+        setup_file="/opt/cpm/bin/sql/setup.sql"
+    fi
```

Comment:
It looks like these two should be different?

```suggestion
    if [[ -f "/pgconf/setup.sql" ]]
    then
        echo_info "Using custom setup.sql"
        setup_file="/pgconf/setup.sql"
    else
        echo_info "Using default setup.sql"
        setup_file="/opt/cpm/bin/sql/setup.sql"
    fi
```

Reply:
Thanks, the directory has been updated.